bin/serve.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:http_server/http_server.dart' show VirtualDirectory;
+import 'package:path/path.dart';
+
+final webroot = join(dirname(dirname(Platform.script.toFilePath())), 'build');
+
+Future<void> main() async {
+  final virDir = new VirtualDirectory(webroot);
+
+  // Set up a directory handler to serve index.html files.
+  virDir.allowDirectoryListing = true;
+  virDir.directoryHandler = (dir, request) {
```

Comment:
Dp we need to do anything to configure the web server so that it doesn't serve outside its root directory?